build/manifest.go

```diff
@@ -45,6 +45,7 @@ type Step struct {
 	Name              string
 	Label             string
 	Dockerfile        string
+  Context           string
```

Comment:
Would you mind running `find . -path ./vendor -prune -type f -o -name '*.go' -exec gofmt -w {} +` and see if it fixes this indentation issue?

Reply:
Oh right, I was not aware of the `gofmt` command, cool!
It is done.